dom.bs

```diff
@@ -1074,12 +1083,14 @@ steps:
  <li><p>Let <var>capture</var> be the result of <a>flattening</a> <var>options</var>.
 
  <li><p>Let <var>once</var> and <var>passive</var> be false.
+ <li><p>Let <var>signal</var> be null.
 
  <li><p>If <var>options</var> is a dictionary, then set <var>passive</var> to <var>options</var>'s
- <code>{{AddEventListenerOptions/passive}}</code> and <var>once</var> to <var>options</var>'s
- <code>{{AddEventListenerOptions/once}}</code>.
+ <code>{{AddEventListenerOptions/passive}}</code>, <var>once</var> to <var>options</var>'s
+ <code>{{AddEventListenerOptions/once}}</code> and <var>signal</var> to
+ {{AddEventListenerOptions/signal}}.
```

Comment:
If we follow my above advice and remove the nullability, then we'll need to break this out into a separate step:

```html
If |options| is a dictionary and |options|["{{AddEventListenerOptions/signal}}"] [=map/exists=], then set |signal| to |options|["{{AddEventListenerOptions/signal}}"].
```

Reply:
It seems this still needs to be done, right? Could also consider nested steps for when options is a dictionary.